### DISCUSSION TITLE
Bug Report: Pagination not change page but add more rows instead

### DISCUSSION THREAD

**Discussion by ThamrinPH** [Power-Components/livewire-powergrid#521] [Ideas]
# PowerGrid Bug Report

#### Have you searched through other issues to see if your problem is already reported or has been fixed?
- [x ] Yes - I did not find it.
- [ ] No

#### Did you read the [documentation](https://livewire-powergrid.com/)?
- [ x] Yes - I did not find it.
- [ ] No

#### Have you tried to publish the views?

You can publish the views to make sure there is no "old code" trapped in views which are not up-to-date.

To publish, run: `php artisan vendor:publish --tag=livewire-powergrid-views`

- [x] Yes - It didn't work.
- [ ] No, this error is not related to views.

#### Is there an error in the console?
- [  ] Yes - I'll add a screenshot or report it below.
- [x] No

### Software Version

| Software  | Version |
|-----------|-------------------|
| PowerGrid |3.2|
| Laravel   |9.19.0|
| Livewire  |2.10|
| Alpine JS  |3|

### Theme
Bootstrap - Version:5

---
## Describe the bug
when i tried to switch page on datatable, instead it change to page 2, it only add new rows

### To Reproduce...

i just click the pagination number or next page button it will add the new items on the table instead switching it

**Screenshots**
From :
![image](https://user-images.githubusercontent.com/24406362/178389320-d3826688-4a0c-4294-bdd0-b6171d980959.png)
To :
![image](https://user-images.githubusercontent.com/24406362/178389421-92c1fe15-0c40-4c22-abef-c564c72b009f.png)

Thank you, i only did the basics and tried some features and meet this bug

**Comment by luanfreitasdev** [comment]
Hello. Which version of bootstrap 5 exactly?

**Comment by ThamrinPH** [comment]
> Hello. Which version of bootstrap 5 exactly?

my mistake i still use bootstrap v4.6.0
but it still persist on bootstrap v5.2.0-beta1

**Comment by luanfreitasdev** [comment]
I believe that this version has some breaks, however only 5.0 is supported.
